crates/forge/tests/data/contracts/deploy_checker.cairo

```diff
@@ -0,0 +1,27 @@
+#[starknet::contract]
+mod DeployChecker {
+    use starknet::ContractAddress;
+
+    #[storage]
+    struct Storage {
+        balance: felt252,
+        caller: ContractAddress,
+    }
+
+    #[constructor]
+    fn constructor(ref self: ContractState, balance: felt252) -> (ContractAddress, felt252) {
+        self.balance.write(balance);
+        self.caller.write(starknet::get_caller_address());
+        (self.caller.read(), balance + 5)
```

Comment:
Is `+5` necessary? 🤔 I don't see the reason why it is added here

Reply:
Just to check return value from the constructor, it doesn't do anything meaningful

Reply:
Ok, so we can return balance. We no not need to increase it